code/modules/mob/living/emote.dm

```diff
@@ -209,9 +209,9 @@
 
 /datum/emote/living/laugh/run_emote(mob/user, params)
 	. = ..()
-	if(. && ishuman(user))
+	if(. && iscarbon(user))
 		var/mob/living/carbon/human/H = user
-		if(H.dna.species.id == "human" && (!H.mind || !H.mind.miming))
+		if((H.dna.species.id == "human" || H.dna.species.id == "felinid") && (!H.mind || !H.mind.miming))

```

Comment:
snowflake and garbage, fix this

Reply:
We have procs for these, not sure why manually checking the variables

Reply:
Theres a is[race]() proc for each race, for example iscatperson(mob) is the one for checking if felinid

Reply:
can we talk about the second check where it asks if you don't have a mind or if you don't have a var on the mind.

Reply:
Is that mind check really necessary :^)